frontend/src/services/list.service.ts

```diff
@@ -29,7 +29,7 @@ const ListService = {
         id: id
       }]
     );
-    const ogData = <IOGData>response.data;

```

Comment:
Is it possible that data is an empty array?

Reply:
It will not return an empty array. Will return this object with null values in case of invalid url name.

[
    {
        "id": null,
        "title": null,
        "description": null,
        "image": null
    }
]